pipelines/institution_id/00-data-assessment-TEMPLATE.py

```diff
@@ -643,6 +768,37 @@
 
 # COMMAND ----------
 
+# degree seeking by enrollment type
+
+(
+    sb.histplot(
```

Comment:
I'm not sure `histplot` is the right plot -- possibly `countplot()` or `barplot()` instead? It looks like this is what we're using elsewhere, though, so presumably I had good reasons for it back when I implemented...

Reply:
I agree, I just defaulted to histplot to follow precedent (which you may have had good reason for)!